packages/dd-trace/test/proxy.spec.js

```diff
@@ -265,15 +265,51 @@ describe('TracerProxy', () => {
 
         let conf = { tracing_enabled: false }
         rc.emit('APM_TRACING', 'apply', { lib_config: conf })
-        expect(appsec.disable).to.have.been.called
-        expect(iast.disable).to.have.been.called
+        expect(appsec.disable).to.not.have.been.called
```

Comment:
should we check 3 lines above, that `appsec.enabled` is not called, in the same way that we are checking in L297 that it is called?
Same in L274

Reply:
This is still missing

> Same in L274

Check that it is not called when apm is enabled by RC.